src/components/InputFile/InputFile.tsx

```diff
@@ -83,12 +94,14 @@ export const InputFile: VFC<Props> = ({
           disabled={disabled}
           tabIndex={-1}
           className={classNames.input}
+          ref={inputRef}
         />
         <FileButton
           themes={theme}
           className={`${FileButtonClassName} ${classNames.button}`}
           disabled={disabled}
           type="button"
+          onKeyDown={(e: React.KeyboardEvent<HTMLButtonElement>) => handleKeyDown(e)}
```

Comment:
[IMO] ここでは引数の型は明示しなくても良いかもと思いました。

```suggestion
          onKeyDown={(e) => handleKeyDown(e)}
```

また、更に簡潔に書くのであれば関数を直接渡して

```suggestion
          onKeyDown={handleKeyDown}
```

としても良いかなと思いましたが、このファイル内の他のハンドラと書き方を合わせている意図であればそれならそれで良いかなとも思います。

Reply:
ありがとうございます！
他のハンドラに合わせる形にした方が統一性があるので前者の方に修正しました！